src/main/java/com/kodilla/ecommercee/CartController/CartController.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.ecommercee.CartController;
+
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartDto;
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartItemDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("v1/carts")
+@RequiredArgsConstructor
+public class CartController {
+
+    @GetMapping("/cart")
```

Comment:
IMHO the path is unnecessarily. I would leave it without a path.

Reply:
Thank you. I'll change it.